bin/api-docs/gen-components-docs/markdown/props.js

```diff
@@ -0,0 +1,53 @@
+function renderPropType( type ) {
+	switch ( type.name ) {
+		case 'enum': {
+			const MAX_ENUM_VALUES = 10;
+			const string = type.value
+				.slice( 0, MAX_ENUM_VALUES )
+				.map( ( { value } ) => value )
+				.join( ' | ' );
+
+			if ( type.value.length > MAX_ENUM_VALUES ) {
+				return `${ string } | ...`;
+			}
+			return string;
+		}
+		default:
+			return type.name;
+	}
+}
+
+function generateMarkdownPropsJson( props, { headingLevel = 2 } = {} ) {
+	const sortedKeys = Object.keys( props ).sort( ( [ a ], [ b ] ) =>
+		a.localeCompare( b )
+	);
+
+	const propsJson = sortedKeys
+		.flatMap( ( key ) => {
+			const prop = props[ key ];
+
+			if ( prop.description.includes( '@ignore' ) ) {
+				return null;
+			}
+
+			return [
+				{ [ `h${ headingLevel + 1 }` ]: `\`${ key }\`` },
+				prop.description,
```

Comment:
This assumes the props will always have a description. Should we add a default description here to handle if that's not the case?

Reply:
When undefined, it should be filtered out by the `.filter( Boolean )` at the end.